autoload/lsp/signature.vim

```diff
@@ -92,9 +92,15 @@ export def SignatureHelp(lspserver: dict<any>, sighelp: any): void
   if sig->has_key('parameters') && sighelp->has_key('activeParameter')
     var params_len = sig.parameters->len()
     if params_len > 0 && sighelp.activeParameter < params_len
-      var label: string = sig.parameters[sighelp.activeParameter].label
-      hllen = label->len()
-      startcol = text->stridx(label)
+      # label = string | [uint, uint]
+      var label = sig.parameters[sighelp.activeParameter].label
+      if label->type() == v:t_string
+        hllen = label->len()
+        startcol = text->stridx(label)
+      else
+        startcol = label[0]
```

Comment:
According to the spec:

The offsets are based on a UTF-16 string representation as `Position` and `Range` does.

So this needs a conversion to a UTF-8 offset.

Reply:
Good point. I'll not have time until next week to look at this. I also think it's worth investigating if [`ClientCapabilities.general.positionEncodings`](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#positionEncodingKind) should be taken into account.